tools/scripts/gen-env-files-java.sh

```diff
@@ -175,3 +175,24 @@ ELASTIC_USERNAME=$elasticUser
 ELASTIC_PASSWORD=$password" >> ./services/java/indexing/.env
     echo "./services/java/indexing/.env created"
 fi
+
+## Transcription service
```

Comment:
Don't add this here.  The `.env` files this script is concerned with are for the java code.

Reply:
I removed that file and put the changes in scripts/gen-env-files.sh yesterday.